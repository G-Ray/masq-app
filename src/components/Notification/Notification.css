.Notification {
	display: flex;
	flex-direction: row;
	justify-content: space-between;
	align-items: center;
	width: 378px;
	height: 40px;
	border-radius: 3px;
	border: solid 1px #e5e5e5;
	margin: auto;
	vertical-align: middle;
}

.Notification p {
	margin-left: 8px;
	color: #707070;
	font-size: 12px;
}

.Notification #left-notif {
	display: flex;
	align-items: center;
	margin-left: 8px;
}

.Notification #infos-icon {
	height: 15px;
	color: #458bf8;
}

.Notification #close-icon {
	height: 9px;
	margin-right: 8px;
	color: #999999;
}
