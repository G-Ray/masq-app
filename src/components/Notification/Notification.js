import React from 'react';

import './Notification.css';

const style = {
  width: '378px',
  height: '40px',
  borderRadius: '3px',
  border: 'solid 1px #e5e5e5',
  margin: 'auto',
  verticalAlign: 'middle',
  display: 'flex',
  flexDirection: 'row',
  justifyContent: 'space-between',
  alignItems: 'center'
};

export default function Notification(props) {
  return (
    <div style={style}>
      <div id="notif">
        <img style={{ height: '15px', marginLeft: '16px', 'marginRight': '16px' }} src='/assets/Infos.svg' alt="infos logo" />
        <p>{props.text}</p>
      </div>
      <img style={{ height: '9px', marginRight: '16px' }} src='/assets/Close.svg' alt="close logo" />
    </div>
  );
}
