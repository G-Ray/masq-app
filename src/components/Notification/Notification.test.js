import React from 'react'
import renderer from 'react-test-renderer'
import Notification from './Notification'

it('renders Notification', () => {
  const component = renderer.create(<Notification text='some notification' />)
  const tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
