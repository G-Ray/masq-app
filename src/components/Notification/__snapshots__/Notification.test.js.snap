// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Notification 1`] = `
<div
  className="Notification"
>
  <div
    id="left-notif"
  >
    <svg
      height={22}
      id="infos-icon"
      viewBox="0 0 22 22"
      width={22}
      xmlnsXlink="http://www.w3.org/1999/xlink"
    >
      <defs>
        <path
          d="M11 0C4.9 0 0 4.9 0 11s4.9 11 11 11 11-4.9 11-11S17.1 0 11 0zm0 20c-5 0-9-4-9-9s4-9 9-9 9 4 9 9-4 9-9 9zm1-9v4c0 .6-.4 1-1 1s-1-.4-1-1v-4c0-.6.4-1 1-1s1 .4 1 1zm-.3-4.7c.2.2.3.4.3.7 0 .3-.1.5-.3.7-.2.2-.4.3-.7.3h-.2c-.1 0-.1 0-.2-.1-.1 0-.1-.1-.2-.1s-.1-.1-.1-.1c-.2-.2-.3-.4-.3-.7 0-.3.1-.5.3-.7l.1-.1c.1 0 .1-.1.2-.1s.1 0 .2-.1c.3 0 .7.1.9.3z"
          id="infos-svg"
        />
      </defs>
      <use
        fill="currentColor"
        fillRule="nonzero"
        xlinkHref="#infos-svg"
      />
    </svg>
    <p>
      some notification
    </p>
  </div>
  <svg
    height={14}
    id="close-icon"
    onClick={undefined}
    viewBox="0 0 14 14"
    width={14}
    xmlnsXlink="http://www.w3.org/1999/xlink"
  >
    <defs>
      <path
        d="M13.7 12.3c.4.4.4 1 0 1.4-.2.2-.4.3-.7.3-.3 0-.5-.1-.7-.3L7 8.4l-5.3 5.3c-.2.2-.4.3-.7.3-.3 0-.5-.1-.7-.3-.4-.4-.4-1 0-1.4L5.6 7 .3 1.7C-.1 1.3-.1.7.3.3c.4-.4 1-.4 1.4 0L7 5.6 12.3.3c.4-.4 1-.4 1.4 0 .4.4.4 1 0 1.4L8.4 7l5.3 5.3z"
        id="close-svg"
      />
    </defs>
    <use
      fill="currentColor"
      fillRule="nonzero"
      xlinkHref="#close-svg"
    />
  </svg>
</div>
`;
